core/geometry/src/test/curve/Arc3d.test.ts

```diff
@@ -144,6 +144,59 @@ describe("Arc3d", () => {
     ck.testCoordinate(arcC.curveLength(), sweepRadians);
     expect(ck.getNumErrors()).toBe(0);
   });
+  it("Arc3dCreate", () => {
+    const ck = new Checker();
+    const allGeometry: GeometryQuery[] = [];
+
+    let dx = 0;
+    const arc = Arc3d.create(
+      Point3d.create(0, 0, 0), Vector3d.unitX(), Vector3d.unitY(), AngleSweep.createStartEndDegrees(0, 7200),
```

Comment:
This is the only line that tests your fix.
1. Consider calling the constructor in the lines below instead of editing the same arc each time.
2. Consider passing more sweeps >360 degrees to some of these constructor calls.